File: libyui-bindings/swig/python/examples/hello_world.py
#
# Trivial libyui example
#

import sys
sys.path.insert(0,'../../../build/swig/python')
import yui

factory = yui.YUI.widgetFactory()
dialog = factory.createPopupDialog()

vbox = factory.createVBox( dialog )
factory.createLabel( vbox, "Hello, World!" )
factory.createPushButton( vbox, "&OK" )
event = dialog.waitForEvent()
dialog.destroy()

File: libyui-bindings/swig/perl/examples/hello_world.pl
#!/usr/bin/perl
#
# Trivial libyui example
#
use lib '../../../build/swig/perl';

use yui;

my $factory = yui::YUI::widgetFactory;
my $dialog = $factory->createPopupDialog;

my $vbox = $factory->createVBox( $dialog );
$factory->createLabel( $vbox, "Hello, World!" );
$factory->createPushButton( $vbox, "&OK" );
my $event = $dialog->waitForEvent();
$dialog->destroy();
